redefinirsenha.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE-edge">
+		<meta name="Viewport" content="width=device-width, initial-scale=1.0">
+		<title>Recuperar Senha</title>
+		<link rel="stylesheet" href="styles.css">
+        <script src="https://kit.fontawesome.com/5cdfb0b317.js" crossorigin="anonymous"></script>
+	</head>
+    <body>
+		<main class="conteiner">
+		<h2> Redefinição de senha</h2>
+       <div class="texto-recuperacao">
+           <p>Para redefinir sua senha, informa o usuario ou e-mail cadastrado na sua conta e lhe enviaremos um link com as instruções.</p>
+       </div> 
+			<form action="">
+				<div class="input-field">
+					<input type="text" nome="username" id="username" placeholder="Digite seu E-mail">
```

Comment:
cuidado com id, se for usar também crie uma label para acomapnhar o input, prefira usar as propridades `aria-`

Reply:
Caso eu não use a label, igual fiz nesse caso apenas com o placeholder, não é necessário o uso do Id então

Reply:
isso aí